themes/gohugo-theme-rene/layouts/portfolio/single.html

```diff
@@ -3,7 +3,12 @@
 {{ end }}
 
 {{ define "main" }}
-  {{ $section := .Site.GetPage "section" .Section }}
+
+  {{ if .Params.redirect }}
+    <meta http-equiv="Refresh" content="0; url='https://expeditions.renehlavova.com/'" />
```

Comment:
Jeste sem muzes pridat:

```suggestion
    <meta http-equiv="Refresh" content="0; url='https://expeditions.renehlavova.com/'" />
    <script type="text/javascript">window.location = "https://expeditions.renehlavova.com/";</script>
```

Ale koukam, ze `meta refresh` je podporovanej vsude, tak je to zbytecny, sorry :D https://caniuse.com/?search=meta%20refresh